src/tests/bundles/bionic-queens.yaml

```diff
@@ -0,0 +1,87 @@
+series: bionic
+relations:
+- - nova-compute:amqp
+  - rabbitmq-server:amqp
+- - keystone:shared-db
+  - mysql:shared-db
+- - nova-cloud-controller:identity-service
+  - keystone:identity-service
+- - glance:identity-service
+  - keystone:identity-service
+- - neutron-api:identity-service
+  - keystone:identity-service
+- - neutron-openvswitch:neutron-plugin-api
+  - neutron-api:neutron-plugin-api
+- - neutron-api:shared-db
+  - mysql:shared-db
+- - neutron-api:amqp
+  - rabbitmq-server:amqp
+- - glance:shared-db
+  - mysql:shared-db
+- - glance:amqp
+  - rabbitmq-server:amqp
+- - nova-cloud-controller:image-service
+  - glance:image-service
+- - nova-compute:image-service
+  - glance:image-service
+- - nova-cloud-controller:cloud-compute
+  - nova-compute:cloud-compute
+- - nova-cloud-controller:amqp
+  - rabbitmq-server:amqp
+- - nova-compute:neutron-plugin
+  - neutron-openvswitch:neutron-plugin
+- - neutron-openvswitch:amqp
+  - rabbitmq-server:amqp
+- - nova-cloud-controller:shared-db
+  - mysql:shared-db
+- - nova-cloud-controller:neutron-api
+  - neutron-api:neutron-api
+- - neutron-api:infoblox-neutron
+  - infoblox:neutron
+
+services:
+  glance:
+    charm: cs:~openstack-charmers-next/glance
+    num_units: 1
+  keystone:
+    charm: cs:~openstack-charmers-next/keystone
+    num_units: 1
+  mysql:
+    charm: cs:~openstack-charmers-next/percona-cluster
+    num_units: 1
+  neutron-api:
+    bindings:
+    charm: /home/ubuntu/dev/infoblox/charms/charm-neutron-api
```

Comment:
The functional test bundle has a hard-coded charm path for neutron-api.  I understand that there is a WIP on that charm, but it is not testable as-is.  Please make your proposed neutron-api charm accessible and deployable (for example, in a cs: namespace) so that this test can be executed.

Reply:
changed to cs:~aym-frikha/charm-neutron-api-0

Reply:
So this needs to become the cs:~openstack-charmer-next/neutron-api once the gerrit review has merged.